Podfile

```diff
@@ -13,3 +13,15 @@ target 'IGListKit-tvOSTests' do
     platform :tvos, '10.0'
     pod 'OCMock', '~> 3.8.1'
 end
+
+post_install do |installer|
+    puts "Running OCMock bitcode workaround. https://github.com/erikdoe/ocmock/issues/475"
+    installer.pods_project.targets.each do |target|
+        target.build_configurations.each do |config|
+            # Disable bitcode for OCMock. See: https://github.com/erikdoe/ocmock/issues/475
+            if target.name.start_with?('OCMock')
+                config.build_settings['ENABLE_BITCODE'] = 'NO'
+              end
+        end
+    end
+end
```

Comment:
Is this the workaruond that fixed the iOS test failure from CI? See my previous attempt that kept failing to open a Pod file: /Users/runner/work/IGListKit/IGListKit/Pods/Target Support Files/OCMock-iOS/OCMock-iOS.debug.xcconfig: unable to open file (in target "OCMock-iOS" in project "Pods") (in target 'OCMock-iOS' from project 'Pods')

https://github.com/Instagram/IGListKit/runs/3345585327?check_suite_focus=true

Reply:
That was my first idea, so I applied this workaround, but it is not needed and I will remove it, thanks. The issue was that you updated OCMock but didn't commit all "new" files.